app/routes/crate/version.js

```diff
@@ -25,14 +28,37 @@ export default class VersionRoute extends Route {
     let version;
     let requestedVersion = params.version_num;
```

Comment:
```suggestion
    let requestedVersion = params.version_num || crate.default_version;
    if (error instanceof NotFoundError) {
      let title = `${crate.name}: Failed to find default version`;
      return this.router.replaceWith('catch-all', { transition, title, tryAgain: true });
    } 
```

I think that might be able to simplify the rest of the code here

Reply:
Hmm, this doesn't seem to align with the current business logic. Currently, we don't fall back to the `default_version` if the user explicitly requests a specific version.

Reply:
I think I get what you're saying.